src/main/java/com/faforever/client/chat/ChatUserItemController.java

```diff
@@ -267,10 +269,13 @@ private void configureCountryImageView() {
 
     countryImageView.setVisible(true);
 
-    Tooltip countryTooltip = new Tooltip(player.getCountry());
-    countryTooltip.textProperty().bind(player.countryProperty());
-
-    Tooltip.install(countryImageView, countryTooltip);
+    final CountryCode countryCode = CountryCode.getByCode(player.getCountry()); //country could be invalid
+    if (countryCode != null) {
+      final Locale countryLocale = new Locale("", player.getCountry());
+      final Tooltip countryTooltip = new Tooltip(countryLocale.getDisplayCountry(i18n.getUserSpecificLocale()));
+      //countryTooltip.textProperty().bind(player.countryNameProperty());
```

Comment:
When not bound to the property, the tooltip will not change when the country of the player changes. So that could be a problem.